src/features/bookOffer/pages/BookingOfferModal.tsx

```diff
@@ -28,11 +34,18 @@ export const BookingOfferModalComponent: React.FC<Props> = ({
     useModalContent(isEndedUsedBooking)
   const enablePricesByCategories = useFeatureFlag(RemoteStoreFeatureFlags.WIP_PRICES_BY_CATEGORIES)
 
-  const modalLeftIconProps = {
-    leftIcon,
-    leftIconAccessibilityLabel,
-    onLeftIconPress,
-  } as ModalLeftIconProps
+  const { height } = useWindowDimensions()
+  const { top } = useCustomSafeInsets()
+  const { modal } = useTheme()
+
+  const { data: offer } = useOffer({ offerId })
+  const stocksWithCategory =
+    offer?.stocks?.filter((stock) => !stock.isExpired && stock.priceCategoryLabel) || []
+  const hasPricesStep = Boolean(stocksWithCategory.length > 1)
+
+  const modalLeftIconProps = useMemo(() => {
+    return { leftIcon, leftIconAccessibilityLabel, onLeftIconPress } as ModalLeftIconProps
+  }, [leftIcon, leftIconAccessibilityLabel, onLeftIconPress])
```

Comment:
L'usage du `useMemo` ici me semble parfaitement inutile.
Tu ne manipules rien alors en l'état ça revient au même que si tu ne mettais pas de useMemo

```suggestion
  const modalLeftIconProps: ModalLeftIconProps = { leftIcon, leftIconAccessibilityLabel, onLeftIconPress }
```

Reply:
Yes à la base il était utilisé dans un useMemo mais j'ai externalisé les composants, je retire